File: Course_01/C Language/EstruturaRepetitiva/dentro_fora.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n, x, countin, countout, i;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);

    countin = 0;
    countout = 0;

    for (i = 0 ; i < n ; i++) {
        printf("Digite um numero: ");
        scanf("%d", &x);
        if (x >= 10 && x <= 20) {
            countin++;
        }
        else {
            countout++;
        };
    }
    printf("%d Dentro\n", countin);
    printf("%d Fora\n", countout);
    return 0;
}

File: Course_01/Python/EstruturasRepetitivas/dentro_fora.py
n = int(input("Quantos números você vai digitar? "))
dentro = 0
for i in range(0, n):
    x = int(input("Digite um número: "))
    if 10 <= x <= 20:
        dentro +=1
print(f"{dentro} DENTRO")
print(f"{n - dentro} FORA")